grants.yml

```diff
@@ -2013,6 +2017,8 @@
     - queue:create-task:highest:adhoc-t/*
     - queue:create-task:highest:scriptworker-prov-v1/{trust_domain}-signing-mac-dev
     - queue:create-task:highest:scriptworker-prov-v1/{trust_domain}-{level}-*
+    - queue:create-task:highest:scriptworker-prov-v1/{trust_domain}-signing-mac14m2*
```

Comment:
Likewise.

Reply:
I got a bit confused since the adhoc worker was both dev and prod, but this makes sense. Moved the prod scope to the master+action grant.